README.md

```diff
@@ -56,6 +56,25 @@ The **arches_her/install/requirements.txt** file will also need to be edited in
 
 
 
+## Running Arches in a Docker Development Environment
+
+You can also run Arches in a Docker development environment. To do this, pull the `arches` repo and use the two compose files `docker-compose-dependencies.yml` and `docker-compose.yml`.
```

Comment:
Looks good. Can you make sure that i refers to running "Arches for HER" rather that just Arches